cms/static/js/factories/import.js

```diff
@@ -87,30 +118,42 @@ define([
                 }
                 if (percentInt >= doneAt) {
                     bar.hide();
-                    // Start feedback with delay so that current stage of import properly updates in session
-                    setTimeout(
-                        function () { Import.startServerFeedback(feedbackUrl.replace('fillerName', file.name));},
-                        3000
-                    );
+
+                    // Start feedback with delay so that current stage of
+                    // import properly updates in session
+                    setTimeout(function () { Import.pollStatus(); }, 3000);
                 } else {
                     bar.show();
                     fill.width(percentVal).html(percentVal);
                 }
             },
-            done: function(event, data){
-                bar.hide();
-                window.onbeforeunload = null;
-                Import.displayFinishedImport();
-            },
-            start: function(event) {
-                window.onbeforeunload = function() {
-                    if (!Import.okayToNavigateAway) {
-                        return "${_('Your import is in progress; navigating away will abort it.')}";
-                    }
-                };
-            },
             sequentialUploads: true,
             notifyOnError: false
         });
+
+
+        var showImportSubmit = function (e) {
+            var filepath = $(this).val();
+
+            if (filepath.substr(filepath.length - 6, 6) === 'tar.gz') {
+                $('.error-block').hide();
+                $('.file-name').html($(this).val().replace('C:\\fakepath\\', ''));
+                $('.file-name-block').show();
+                chooseBtn.hide();
+                submitBtn.show();
+                $('.progress').show();
+            } else {
+                $('.error-block').html(gettext('File format not supported. Please upload a file with a <code>tar.gz</code> extension.')).show();
```

Comment:
I know that you didn't change this message, but seeing it in this pull request, it seems like it could be better internationalized to reduce the likelihood of a non-technical translator trying to translate the `<code>` or `tar.gz` in the string. Would you mind doing something like this, instead?
```js
var msg = gettext('File format not supported. Please upload a file with a {file_extension} extension.')
    .replace('{file_extension}', '<code>tar.gz</code>');
$('.error-block').html(msg).show();
```
If you'd rather not roll this change into your pull request, because you don't want it to get too big, just let me know and I can make a follow-on pull request after this one merges.